specifyweb/specify/record_merging.py

```diff
@@ -15,13 +15,31 @@
 
 from specifyweb.businessrules.exceptions import BusinessRuleException
 from specifyweb.celery_tasks import LogErrorsTask, app
+from specifyweb.accounts import models as acccounts_models
+from specifyweb.attachment_gw import models as attachment_gw_models
+from specifyweb.businessrules import models as businessrules_models
+from specifyweb.context import models as context_models
+from specifyweb.notifications import models as notifications_models
+from specifyweb.permissions import models as permissions_models
+from specifyweb.interactions import models as interactions_models
+from specifyweb.workbench import models as workbench_models
 from . import api, models as spmodels
 from .api import uri_for_model
 from .build_models import orderings
 from .load_datamodel import Table, FieldDoesNotExistError
 from celery.utils.log import get_task_logger # type: ignore
 logger = get_task_logger(__name__)
 
+
+APP_MODELS = [spmodels, acccounts_models, attachment_gw_models, businessrules_models, context_models,
```

Comment:
does APP_MODULES & get_app_model variable belong in this file, or should it be moved into some more central place?

Reply:
ya, there might be a better place for this.